adal/src/androidTest/java/com/microsoft/aad/adal/BasicWebViewClientTests.java

```diff
@@ -83,7 +73,7 @@
     private static final String TEST_PLAY_STORE_INSTALL_AUTH_APP_URL =
             PLAY_STORE_INSTALL_PREFIX + AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME;
     private static final String TEST_PLAY_STORE_INSTALL_CP_URL =
-            PLAY_STORE_INSTALL_PREFIX + COMPANY_PORTAL_PROD_APP_PACKAGE_NAME;
```

Comment:
I'd assume that we will need the actual (PROD) package name for both authapp and CP for this? 

@tanmaymanolkar1 how does these test cases work?

Reply:
With my changes there is now only one Company Portal and this is the actual prod CP....I've basically renamed the constants... the other one is actually called BROKER_HOST_PACKAGE_NAME